cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PackCommand.java

```diff
@@ -177,6 +179,23 @@ public void execute() {
             if (buildOptions.dumpBuildTime()) {
                 BuildTime.getInstance().projectLoadDuration = System.currentTimeMillis() - start;
             }
+
+            Path packageMdPath = this.projectPath.resolve("Package.md");
```

Comment:
After the Swan Lake Update 11, the default file is `README.md`. For backward compatibility, we support `Package.md` too. Also, the user can set a custom file in the `Ballerina.toml` with the `readme = "<md-file-path>"` property. 

Therefore, we should use the `project.currentPackage().manifest().readme()` to get the README file. The path can be relative to the project or it can be an absolute path as well.